threedi_modelchecker/threedi_model/constants.py

```diff
@@ -1,10 +1,11 @@
 from enum import Enum
 
 
-LATEST_MIGRATION_ID = 173
-LATEST_MIGRATION_NAME = (
-    "0171_auto__chg_field_v2aggregationsettings_aggregation_method__del_field_v2"
-)
+LATEST_SOUTH_MIGRATION_ID = 174
+#  Migration 174 deletes some fields, which were already not present in the
```

Comment:
Good, we need a Django migration worker (for a period of time) to get to migration 174 for users that have schematisations with older migrations. Think we can add a better (error) message with instructions what to do in the functions/libs that are going to be build upon this lib.